### PR TITLE
Added serialization of product in website-controller

### PR DESCRIPTION
**Pull request by wachterjohannes**

### CODE REVIEW

File: Controller/Product/WebsiteProductController.php [hunk 1]
```diff
@@ -29,16 +30,46 @@ class WebsiteProductController implements ContainerAwareInterface
...
 
+    protected function getAttributes(array $attributes, ProductViewInterface $object): array
+    {
+        return $this->getAttributeResolver()->resolve(array_merge($this->serialize($object), $attributes));
+    }
+
+    protected function serialize(ProductViewInterface $object): array
+    {
+        /** @var array $result */
+        $result = $this->getSerializer()->serialize($object, 'array');
```
Comment (trickreich) [inline Controller/Product/WebsiteProductController.php]: var needed?

File: Controller/Product/WebsiteProductController.php [hunk 2]
```diff
@@ -29,16 +30,46 @@ class WebsiteProductController implements ContainerAwareInterface
...
+
+        return $result;
+    }
+
+    protected function getSerializer(): SerializerInterface
+    {
+        /** @var SerializerInterface $serializer */
+        $serializer = $this->get('jms_serializer');
```
[the remark this answers is not in the document]
Comment (wachterjohannes) [reply Controller/Product/WebsiteProductController.php]: yes - else PHPStan will cry :)

File: Controller/Product/WebsiteProductController.php (hunk 1 above)
Comment (wachterjohannes) [reply Controller/Product/WebsiteProductController.php]: yes - else PHPStan will cry :)

File: Resources/config/serializer/Product.ProductView.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<serializer>
+    <class name="Sulu\Bundle\SyliusConsumerBundle\Model\Product\ProductView" exclusion-policy="ALL">
+        <virtual-property method="getCode" serialized-name="uuid" expose="true"/>
```
Comment (trickreich) [inline Resources/config/serializer/Product.ProductView.xml]: code is uuid?

Comment (wachterjohannes) [reply Resources/config/serializer/Product.ProductView.xml]: should be id and code now

thanks for the hint

**Pull request merged**